docs/configuration/intra_node_data_path.md

```diff
@@ -183,3 +190,75 @@ On the other hand, CARET requires users to provide the following description if
 
 User have to fill callback name in `variable_passings`, `publishes`'s `callback_name`. `context_type` must be set as `callback_chain`.
 After editing, use path.verify() described in the beginning of this section to verify that it has been set correctly.
+
+### Manipulating the object
```

Comment:
```suggestion
### Manipulating the Architecture object
```

Reply:
There is a concern that the names of the sub-sections displayed in the `Table of contents` will become too long.